### DISCUSSION TITLE
Filter Query by date

### DISCUSSION THREAD

**Discussion by Serhiy-Nazarov** [motor-admin/motor-admin#71] [General]
Hello there! Could you help me to create query with dynamic filter by date. It should looks such as filter by ID.
For example,

```
SELECT * FROM "billings"
WHERE "billings"."id" = {{billing_id}}
```
In this case I'll see the dropdown filter by billing_id

But if I do this one
```
SELECT * FROM "billings"
WHERE "billings"."date" = {{billing_date}}
```
In this case I can see only text input without dropdown. Is it possible to make dropdown for select date from the list?

Thanks!

**Comment by omohokcoj** [comment]
@Serhiy-Nazarov hey hope this helps:
<img width="1015" alt="image" src="https://user-images.githubusercontent.com/5418788/207832084-3f86e56d-24cb-4323-a99b-63e4e18094ce.png">

**Reply by Serhiy-Nazarov** [reply]
@omohokcoj it's great! Thanks!
But I want to have dropdown with a list of dates. Not datepicker.
If I select "reference" input type I can't select as a source a different query. Do you have any idea?

**Reply by omohokcoj** [reply]
@Serhiy-Nazarov currently the only way to achieve something similar is by using the predefined selector values:
![image](https://user-images.githubusercontent.com/5418788/207953453-21082bc9-e109-4572-9c18-ad98b98181e8.png)
Those selector values can be converted to timeframes in SQL - for now it's not possible to populate selector values from other query.
